reuse.go

```diff
@@ -0,0 +1,190 @@
+package libp2pquic
+
+import (
+	"errors"
+	"net"
+	"sync"
+
+	srcs "github.com/lnykww/go-src-select"
+)
+
+type ReuseConn struct {
+	net.PacketConn
+	lock sync.Mutex
+	ref  int
```

Comment:
Why is `ref` an `int`, if it only ever takes the value `0` or `1`?

Reply:
ref indicates how many connection was use it. and its value is not only  0 or 1.